### PR TITLE
Landing page v1

### LINKED ISSUE #31: Horizontal partner list
Bildschirmfoto 2022-12-14 um 7 07 14 PM

### PR DESCRIPTION
**Pull request by JannikWibker**
Closes #31

### CODE REVIEW

**Comment by JannikWibker** [comment]
image

**Comment by JannikWibker** [comment]
What this PR does at the moment:
- move the landing page to also use [twind](https://twind.style)
- clean up colors a little bit
- add carousel (left side), doesn't have any actual functionality and is purely visual right now
- add "feature X" checkboxes
- add color to "newly" and "rekindled" words

**Comment by MaxSchaefer** [comment]
@JannikWibker please make sure you use Husky to comply with our commit conventions. In the case of this pr, a squash should work :)

**Comment by JannikWibker** [comment]
what's still missing:
- features section
- roadmap section
- start section ui screenshots
- proper contact us form with api hubspot integration
- smooth scroll animation for carousel
- linear gradients for `<GridBox />`
- styling improvements here and there
- align header items with carousel items

**Review by use-to — COMMENTED** [review]
I would discuss some of the changes!
But from a design perspective: no objections

File: landing-page/components/sections/ContactSection.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  const handleSubmit = (e: FormEvent) => {
+    e.preventDefault()
+    console.log({
+      email,
+      firstName,
+      lastName,
+      message,
+    })
+  }
```
[the remark this answers is not in the document]
Comment (MaxSchaefer) [reply landing-page/components/sections/ContactSection.tsx:22]: Yes, we can submit the form via the [HubSpot Api](https://legacydocs.hubspot.com/docs/methods/forms/submit_form). But for now, we should revert the changes on the contact form and proceed with the current working (iframe) form. We should not take a step back on a great design but without functionality. I can intergrate the api with #36 in the future.

@JannikWibker can you revert the changes on the contact form and move those to a separate branch?

[the remark this answers is not in the document]
Comment (use-to) [reply landing-page/components/sections/ContactSection.tsx:22]: Moved to `ignore/redesign-contact-section`! So we can use these changes later on...

[the remark this answers is not in the document]
Comment (use-to) [reply landing-page/components/sections/ContactSection.tsx]: Moved to issue #44
Draft pull-request #45

**Review by use-to — APPROVED** [review]
No further concerns up to this point from my side! 🚀

**Comment by use-to** [comment]
Respectable down to a device width of 900px...

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**